### PR TITLE
Some corrections and thoughts

### PR DESCRIPTION
**Pull request by leletec**

### CODE REVIEW

File: paper.md [hunk 1]
```diff
@@ -531,3 +531,5 @@ escapen.
  - IETF
  - The Go Authors
  - sam
+
+ [momentane Seitenzahl in korrekter Formatierung ab 1.: 14.0]
```
Comment (okitec) [inline paper.md]: Thanks for that information.

File: paper.md [hunk 2]
```diff
@@ -329,7 +329,7 @@ dass einige der `+JAWOHL`s noch fehlen. Außerdem sieht man den einzigartigen
 erste Argument nennt, hier `1`. Durch einen Mitschnitt des Protokolls ab dem
 Beginn kann man den gesamten Verlauf des Spiels verfolgen.
 
-	S: +JAWOHL Willkommen, Genosse! Subscriben Sie!
+	S: +JAWOHL Willkommen, Genosse! Subscriben Sie![so happy about that]
```
Comment (okitec) [inline paper.md]: Always a pleasure, Genosse!

File: paper.md [hunk 3]
```diff
@@ -286,7 +286,7 @@ verwalten; dadurch kann man von mehreren Geräten auf denselben Baum zugreifen.
 Der Client cacht die Mails nur; der Server hat die relevante Kopie.
 Verständlicherweise ist IMAP komplexer als POP3.
 
-Ich will IMAP deswegen ansprechen, weil es *Tags* verwendet, wie es auch das
+Ich["zu vermeiden", in dem Fall aber wohl sinnvoll] will IMAP deswegen ansprechen, weil es *Tags* verwendet, wie es auch das
```
Comment (okitec) [inline paper.md]: Ich schreibe, wie ich will.

File: paper.md [hunk 4]
```diff
@@ -259,7 +259,7 @@ hier nicht. Newsseiten sind häufige Übeltäter.
 
 HTTP ist nicht für bidirektionale Kommunikation gedacht, da die
 Zustandslosigkeit im Weg steht. Cookies sind ein Hack, um diese zu umgehen, und
-niemand mag Cookies. Ein anderer Weg sind Parameter in der URL, die den ganzen
+niemand mag Cookies[lol]. Ein anderer Weg sind Parameter in der URL, die den ganzen
```
Comment (okitec) [inline paper.md]: Niemand mag Web-Cookies.

File: paper.md [hunk 5]
```diff
@@ -211,19 +211,19 @@ einer *Era Number*, einem in Sekunden gemessenen *Era Offset* und einem Bruch
...
 deswegen wird UDP auf Port 123 verwendet. NTP verwendet konventionelle binäre
 Pakete mit einem Header und einem aus vier Timestamps bestehenden Payload. Ein
-invalider Wert im Header, Stratum 0, initiiert ein *Kiss-o'-Death*-Paket, mit
+invalider Wert im Header, Stratum 0, initiiert ein *Kiss-o'-Death*-Paket[warum der Name], mit
 welchem Kontrollcodes übertragen werden; diese sind Vier-Zeichen-ASCII-Strings
 an der Stelle, an der sonst die Referenz-ID des Zeitgebers steht (z.B. "GPS").
 
 
-XXX mehr Fokus auf Protokoll, weniger auf Umstände?
+XXX mehr Fokus auf Protokoll, weniger auf Umstände?[Ich fand die Umstände interessant]
```
Comment (okitec) [inline paper.md]: Alright, thanks for the feedback.

File: paper.md [hunk 6]
```diff
@@ -211,19 +211,19 @@ einer *Era Number*, einem in Sekunden gemessenen *Era Offset* und einem Bruch
...
-Sekundenzähler und einen Bruch; das *Short*-Format ist ähnlich, hat aber nur
+Sekundenzähler und einen Bruch[für Millisekunden? nicht ganz klar]; das *Short*-Format ist ähnlich, hat aber nur
 16 Bit Präzision.
 
 TCP kann hier nicht verwendet werden, weil es verlorene Pakete wieder
 überträgt und dadurch die Zeitstempel in diesen verfälscht [citation needed],
 deswegen wird UDP auf Port 123 verwendet. NTP verwendet konventionelle binäre
 Pakete mit einem Header und einem aus vier Timestamps bestehenden Payload. Ein
-invalider Wert im Header, Stratum 0, initiiert ein *Kiss-o'-Death*-Paket, mit
+invalider Wert im Header, Stratum 0, initiiert ein *Kiss-o'-Death*-Paket[warum der Name], mit
```
Comment (okitec) [inline paper.md]: The protocol creators are hackers and chose to use this well-established term for control packets.

File: paper.md [hunk 7]
```diff
@@ -211,19 +211,19 @@ einer *Era Number*, einem in Sekunden gemessenen *Era Offset* und einem Bruch
 besteht. Die *Era Number* bezeichnet den Bereich, in dem der 32-bit Offset nicht
 überläuft. Momentan sind wir in Era 0; ab dem 08. Februar 2036 werden wir in
 Era 1 sein. Das im Protokoll verwendete *Timestamp*-Format hat einen 32-bit
-Sekundenzähler und einen Bruch; das *Short*-Format ist ähnlich, hat aber nur
+Sekundenzähler und einen Bruch[für Millisekunden? nicht ganz klar]; das *Short*-Format ist ähnlich, hat aber nur
```
Comment (okitec) [inline paper.md]: Das werd ich wohl noch präzisieren müssen; wusst ich auch nicht besser und habs grade anhand von Beispielcode herausgefunden. Naja, die Auflösung des Fraktionsteils hängt von der Anzahl an Bits ab. Man stelle sich das als Festkommazahl vor: 16 Bit vor dem Komma, 16 Bit danach.

File: paper.md [hunk 8]
```diff
@@ -417,7 +417,7 @@ selektiert das eine Buch mit dieser ISBN und gibt alle Informationen darüber
 aus. Im zweiten Beispiel werden alle Copies selektiert, die die User `Hans` und
 `Max Mustermann` ausgeliehen haben; diese werden zurückgegeben (`r`) und dann
 ganz aus dem System gelöscht (`d`), weil Hans und Max eine Bücherverbrennung
-veranstaltet haben. Das dritte Beispiel selektiert die Ausleiher der Copies mit
+veranstaltet haben[omg]. Das dritte Beispiel selektiert die Ausleiher der Copies mit
```
Comment (okitec) [inline paper.md]: serious business

**Comment by okitec** [comment]
Thanks for your values input. I've fixed the typos and noted down some changes I shall do in the future. However, I won't merge the PR because I would have to remove all markings I disregarded.

**Comment by leletec** [comment]
No problem.
This was not intended to be merged, but the easiest way to let you know about my changes.

**Comment by okitec** [comment]
That's what I had guessed. :smile:

**Comment by leletec** [comment]
I know, just quoting Mr. P

**Comment by okitec** [comment]
How exactly is that a quote? I don't really get it.

**Comment by leletec** [comment]
That was supposed to be an answer to the thing that you should not use "I"

**Comment by okitec** [comment]
I see. You failed. *drops mic*

**Pull request closed without merging**